RecoTracker/MkFitCore/interface/binnor.h

```diff
@@ -176,7 +176,9 @@ namespace mkfit {
 
     // Access
 
-    B_pair m_bin_to_n_bin(B_pair m_bin) { return {m_a1.M_bin_to_N_bin(m_bin.bin1()), m_a2.M_bin_to_N_bin(m_bin.bin2())}; }
+    B_pair m_bin_to_n_bin(B_pair m_bin) {
+      return {m_a1.M_bin_to_N_bin(m_bin.bin1()), m_a2.M_bin_to_N_bin(m_bin.bin2())};
```

Comment:
another place to lower the case 
```suggestion
      return {m_a1.m_bin_to_N_bin(m_bin.bin1()), m_a2.m_bin_to_N_bin(m_bin.bin2())};
```